Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -0,0 +1,160 @@
+
+
+## Editing Blocks
+
+When viewing a **Block List** editor for the first time, you'll be presented with the option to Add content
+
+![Block List - Add Content](images/BlockListEditor_AddContent.png)
+
+Clicking the Add content button brings up the Block Catalogue
+
+![Block List - Simple setup](images/BlockListEditor_BlockPicker_simplesetup.jpg)
+
+The Block Catalogue looks different depending on the amount of avaiable Blocks and their catalogue apperance.
+
+![Block List - example setup from Umbraco.com](images/BlockListEditor_BlockPicker.jpg)
+
+Click the Block type you wish to create and a new Block will appear in the list.
+
+Depending on wether your Block List Editor is setup to use default or inline editing mode.
+In default mode you will enter the editing overlay of that Block:
+
+![Block List - simple overlay editing](images/BlockListEditor_EditingOverlay.jpg)
+
+In inline editing mode the new Blocks inline editor will expand:
+
+![Block List - simple inline editing](images/BlockListEditor_InlineEditing.jpg)
+
+
+More Blocks can be added to the list by clicking the Add content button or using the inline Add content button that appears on hover between or above existing Blocks.
+
+![Block List - Add Content](images/BlockListEditor_AddContentInline.jpg)
+
+To reorder the Blocks, click and drag a Block up or down to place in the decirede order.
+
+To delete a Block click the trash-bin icon appearing on hover.
+
+## Rendering Block List Content
+
+Rendering the stored value of your **Block List** property can be done in two ways.
+
+### 1. Default rendering
+Use the build-in rendering machinsm for rendering blocks via a Partial View for each block.
+
+To make this work you will need to create a Partial View for each block, named by the alias of the ElementType thats begin used as Content Model.
+These partial views must be placed in this folder: `Views/Partials/BlockList/Components/`.
+
+A Partial View will receive the model of `Umbraco.Core.Models.Blocks.BlockListItem`. This gives you the option to access properties of the Content and Settings section of your Block.
+
+In the following exmalpe please note that the `ElementTypeAliasOfContent`and `ElementTypeAliasOfSettings` should corrospond with the selected ElementType Alias for the given model in your case.
+
+Example:
+
+```csharp
+@inherits Umbraco.Web.Mvc.UmbracoViewPage<Umbraco.Core.Models.Blocks.BlockListItem>
+@using ContentModels = Umbraco.Web.PublishedModels;
+@{
+    var content = (ContentModels.ElementTypeAliasOfContent)Model.Content;
+    var settings = (ContentModels.ElementTypeAliasOfSettings)Model.Settings;
+}
+```
+
+### 2. Build your own rendering
+A built-in value converter is available to use the data as you like. Call the `Value<T>` method with a generic type of `IEnumerable<BlockListItem>` and the stored value will be returned as a list of `BlockListLayoutReference` entities.
```

Comment:
If you're calling the Value<T> method with a type, you're going to get a result of that type. So I don't really see why the returned list would contain `BlockListLayoutReference` entities?

Reply:
It will not, this has been updated.